app/tailwind.css

```diff
@@ -591,6 +591,14 @@ video {
   }
 }
 
+.absolute {
+  position: absolute;
+}
+
+.relative {
+  position: relative;
+}
+
```

Comment:
1. Why is there a need to define these classes? Ideally Tailwind's utilities must be used
2. Is this the correct way of extending Tailwind styles?

Reply:
tailwind classes are automatically generated by a script.

Reply:
Got it